### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/feature/code-block.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#871] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/feature/code-block.html

高亮器 Shiki 使用 @vuepress/plugin-shiki，你可以使用 shiki 来高亮你的代码块。 相关信息 vuepress-theme-hope 将主题选项中的 plugins.shiki 选项作为插件选项提供给 @vuepress/plugin-shiki。 代码主题 如果你想要使用单一主题，你可以在主题选项中设置 plugins...

https://theme-hope.vuejs.press/zh/guide/feature/code-block.html

**Comment by Noachlu** [comment]
shiki: {
      lineNumbers: true,
      themes: {
        light: "one-light",
        dark: "one-dark-pro",
      },
    },
为什么lineNumbers: true,用来显示代码块行号，没有效果

**Reply by Noachlu** [reply]
是只能进行主题的配置吗？？
